tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -168,7 +180,20 @@ def _run(self):
                                                 'volume%s.brick%s.mount_options' % (
                                                     index, b_index))
                                         )
-                                    NS.brick.save()
+                                    # Store the bricks sequentially, as the order of bricks
+                                    # matter while figuring out the sub-volumes
```

Comment:
Whats the reason to overload NS with NS.brick or NS.volume here? 

If there is no important reason, Id prefer we dont overload NS in this file at all. Please use local variables for brick, volume, vol_opts etc which are assigned NS right now